packages/esm-bed-management-app/src/config-schema.ts

```diff
@@ -0,0 +1,10 @@
+import { Type } from "@openmrs/esm-framework";
+
+export const configSchema = {
+  admissionLocationTagUuid: {
+    _type: Type.UUID,
+    _description:
+      "UUID for the location tag of the `Admission Location`. Patients may only be admitted to inpatient care in a location with this tag",
+    _default: "1c783dca-fd54-4ea8-a0fc-2875374e9cb6",
```

Comment:
No location tag `Admission Location` was found on the dev3
@ibacher, do we need to add a new tag in the Distro RefApp?
Thanks!

Reply:
It would be best if we could broaden this to support a `uuid or name` of a location tag, and then configure this by default to be `Admission Location`, which is what is expected throughout the backend modules, including EMR API and Bed Management.  I do not support hard-coding uuids as defaults in esm configs in general.

Reply:
Agreed, this property should be something like "admission location tag" instead of "admission location uuid" and default to "Admission Location"

Reply:
Yeah I would love it if we could reference things by name in the configs, but I'm not aware of the location endpoint will query locations by location tag name. If that is possible, it's not documented in the [REST docs](https://rest.openmrs.org/#location). Though it should be noted that neither is getting locations by location tag UUID documented there. Not sure if there's some other source of documentation or people just read the backend Java code to figure out what the API is.

The other possibility is to support using location tag name in the config by making two requests. First we would get all the location tag data from `/locationtag`, then look up the UUID by the configured name, then use that UUID in the `/location` query.

Reply:
Well, the REST docs are incomplete on that point; we [support both](https://github.com/openmrs/openmrs-module-webservices.rest/blob/57b2e5ed9ce59201a351093ea68e62de408799ff/omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/LocationSearchHandler.java#L62).

Reply:
Oh I replaced the UUID with a tag name and it just works :laughing:  We should probably document this somewhere

Reply:
TBH, in the future my hope is that we don't need this Admission Location tag at all.  I don't really see why the bedmanagement module should restrict which locations beds can be assigned to to only "Admission Location" locations.  There really is no need for that, and in fact it may be that we want to support assigning patients to beds without actually admitting them, as was indicated in a recent slack conversation...  But that's for a future ticket.

Reply:
> If we remove the tag, how will we distinguish locations which are not wards? We still want a way to mark a location as admission location since not every location can admit patients

I'm not saying we should remove the tag from the back end, or from use in appropriate places.  But all we are aiming to harvest here is a UI for creating beds and assigning beds to a location, as well as managing bed types and tags.  Whether or not the locations we are assigning beds to are considered inpatient wards that can admit patients is not necessarily a factor.

Reply:
For now, I think Admission Location is used on the backend as well, so this should stay as-is.  I was just commenting on the fact that this might go away in the future.  A new location tag would be fine if we need to distinguish.  Personally, I see no harm in an admin UI for adding beds to locations broadly.  Then, the locations that users are allowed to assign patients to beds are those locations that have beds allocated to them...